### ISSUE TITLE
validation accuracy not changing much

### ISSUE THREAD

**Issue by resplendentfrever** [fchollet/keras#2367]
Hi,

I am using a LSTM network for sentiment classification. I ran it for 100 epochs and then plotted the training and validation accuracy. The training accuracy curve shows a rise but there is not much change in the validation accuracy.
I have done 10 cv by taking 10 different validation sets and running the model from the scratch.
Is something going wrong in my model?

**Comment by ersinyar** [comment]
@resplendentfrever I am experiencing a similar issue. In my case val_acc fluctuates in a small interval. Also it does not start from a low value. Have you come up with a solution yet? If so, how?
